scripts/quickstart_spm_xcodeproj.sh

```diff
@@ -0,0 +1,29 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Modify a .xcodeproj to use a specific branch.
+# TODO: Update to transform from a release, as well as from `main`.
+
+set -xeuo pipefail
+
+SAMPLE=$1
+XCODEPROJ=${SAMPLE}/${SAMPLE}Example.xcodeproj/project.pbxproj
+
+if grep -q "branch = main;" ${XCODEPROJ}; then
```

Comment:
nit: Double quote to prevent globbing and word splitting. -- [SC2086](https://www.shellcheck.net/wiki/SC2086)

```suggestion
if grep -q "branch = main;" "$XCODEPROJ"; then
```